src/descriptive.js

```diff
@@ -29,22 +29,24 @@ export function max(arr) {
 }
 
 /**
- * Return the sum of all entries in a numeric array.
+ * Return the sum of all entries in a numeric array,
+ * using the Kahan summation algorithm.
+ * This compensates for loss of precision when the result is large compared to
+ * the smallest items in the array.
+ * See https://en.wikipedia.org/wiki/Kahan_summation_algorithm
```

Comment:
Please remove. Not necessary as the name of the algorithm is reference and the doc provides a link to the wikipedia.